graphbolt/CMakeLists.txt

```diff
@@ -178,30 +221,40 @@ if(USE_CUDA)
   target_link_libraries(${LIB_GRAPHBOLT_NAME} ${LIB_GRAPHBOLT_CUDA_NAME})
 
 elseif(USE_HIP)
-  file(GLOB BOLT_CUDA_EXTENSION_SRC
+  file(GLOB LIB_GRAPHBOLT_HIP_SRC
     ${BOLT_DIR}/cuda/extension/*.cu
     ${BOLT_DIR}/cuda/extension/*.cc
     ../third_party/HugeCTR/gpu_cache/src/nv_gpu_cache.cu
   )
-  add_library(${LIB_GRAPHBOLT_HIP_NAME} STATIC ${BOLT_CUDA_EXTENSION_SRC} ${BOLT_HEADERS})
-  target_link_libraries(${LIB_GRAPHBOLT_HIP_NAME} "${TORCH_LIBRARIES}")
+  message(STATUS "LIB_GRAPHBOLT_HIP_SRC: ${LIB_GRAPHBOLT_HIP_SRC}")
```

Comment:
I assume this is leftover debugging? Would be nice to include a status message mirroring the USE_CUDA branch